pyre.py

```diff
@@ -47,9 +51,46 @@ def color(r, g, b):
     assert(len(self.particles) == self.NUM_PARTICLES)
 
     self.resize()
+    self.volume = 1.0
+    if pygame_available:
```

Comment:
That would require threading to be imported when pyaudio isn't being used, which seems unnecessary.